src/Bicep.Core/Analyzers/Linter/LinterAnalyzer.cs

```diff
@@ -22,18 +21,16 @@ public class LinterAnalyzer : IBicepAnalyzer
         public const string AnalyzerName = "core";
         public static string LinterEnabledSetting => $"{SettingsRoot}:{AnalyzerName}:enabled";
         public static string LinterVerboseSetting => $"{SettingsRoot}:{AnalyzerName}:verbose";
-
-        private readonly ConfigHelper defaultConfigHelper = new ConfigHelper();
         private ConfigHelper activeConfigHelper;
         private ImmutableArray<IBicepAnalyzerRule> RuleSet;
         private ImmutableArray<IDiagnostic> RuleCreationErrors;
 
         // TODO: This should be controlled by a core component, not an analyzer
         public const string FailedRuleCode = "linter-internal-error";
 
-        public LinterAnalyzer()
+        public LinterAnalyzer(ConfigHelper configHelper)
         {
-            this.activeConfigHelper = this.defaultConfigHelper;
+            this.activeConfigHelper = configHelper;
```

Comment:
[nit] rename `activeConfigHelper` to just `configHelper`

Reply:
Updated as part of this commit:
https://github.com/Azure/bicep/commit/1e2de9b409ea393d9b1c034e9f615d5cf1bc3ca6